crates/shared/src/test_utils/node_url.rs

```diff
@@ -8,14 +8,8 @@ use url::Url;
 /// - `node_rpc_url()` -> <https://example.com/rpc/v0_7>
 /// - `node_url()` -> <https://example.com/>
 pub fn node_rpc_url() -> Result<Url> {
-    dotenv::dotenv().ok();
-    let node_rpc_url = env::var("SEPOLIA_RPC_URL")
-        .context("The required environmental variable `SEPOLIA_RPC_URL` is not set. Please set it manually or in .env file"
-    )?;
-
-    Url::parse(&node_rpc_url).with_context(|| {
-        format!("Failed to parse the URL from the `SEPOLIA_RPC_URL` environmental variable: {node_rpc_url}")
-    })
+    Url::parse("http://188.34.188.184:7070/rpc/v0_7")
+        .with_context(|| "Failed to parse the sepolia RPC URL")
```

Comment:
`.unwrap()` here would be better, it should not fail

Reply:
Hmm i think it's true but i prefer to have that in case somebody changes the URL - it's easier to debug then if the URL is incorrect

Reply:
I can unwrap it probably here though after the context